politeiawww/politeiawww.go

```diff
@@ -112,6 +112,7 @@ type politeiawww struct {
 	test bool
 
 	// Following entries require locks
+	// XXX userPubkeys can be removed now that the userdb is queryable
```

Comment:
we do want to remain compat with leveldb so while it can go away we may have to move this functionality into the db (e.g. make it queriable and maybe add an index)

Reply:
I agree.